### ISSUE TITLE
update 20 usages of Arrays.asList

### ISSUE THREAD

**Issue by ashtonhogan** [yegor256/cactoos#769]
This issue is a child of https://github.com/yegor256/cactoos/issues/62 and should address calls like those seen [here](https://github.com/yegor256/cactoos/blob/b4223ce3e096424397e33058568b587844d93b24/src/main/java/org/cactoos/iterable/IterableOf.java#L50) with a non-static alternative

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by llorllale** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/yegor256/cactoos/issues/769#issuecomment-382105078))

@llorllale Job [#769](https://github.com/yegor256/cactoos/issues/769) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @ashtonhogan[/z](https://www.0crat.com/u/ashtonhogan)

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@9c0ff7e3-e0b9-4bc8-930d-80f60a6be371, thread: BkParallel-3
com.amazonaws.http.AmazonHttpClient$RequestExecutor[1113] com.amazonaws.SdkClientException: Unable to execute HTTP request: null
sun.nio.ch.FileChannelImpl[110] java.nio.channels.ClosedChannelException: null
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by 0crat** [comment]
The job [#769](https://github.com/yegor256/cactoos/issues/769) assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/C63314D6Z/1222147224); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by llorllale** [comment]
@0crat resign @Vatavuk

**Comment by llorllale** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat resign @Vatavuk  ([here](https://github.com/yegor256/cactoos/issues/769#issuecomment-386413810))

@llorllale The user @vatavuk[/z](https://www.0crat.com/u/vatavuk) resigned from [#769](https://github.com/yegor256/cactoos/issues/769), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/cactoos/issues/769#issuecomment-386413841))

@llorllale The job [#769](https://github.com/yegor256/cactoos/issues/769) is now out of scope

**Comment by llorllale** [comment]
@ashtonhogan fix merged in #769, can we close this?

**Comment by 0crat** [comment]
The job is not in WBS, won't close the order
